ui/partials/perf/comparerror.html

```diff
@@ -0,0 +1,2 @@
+<div class="alert alert-danger" role="alert" ng-repeat="error in errors"><span class="fa fa-warning"></span> {{error}}</div>
+<a href="#/comparechooser?originalProject={{originalProject}}&originalRevision={{originalRevision}}&newProject={{newProject}}&newRevision={{newRevision}}">Modify choice</a>
```

Comment:
A few things here:

1. Please seperate out each html tag into multiple lines.
2. Let's encapsulate the text and link content into a single alert. You should be able to do this by laying things out like this:

```html
    <div class="alert alert-danger" role="alert">
      <p ng-repeat="error in errors">
        <span class="fa fa-warning"></span> {{error}}
      </p>
      <p><a class="alert-link" href="..."></p>
    </div>
```

The end result should look something like this:

<img width="645" alt="screen shot 2015-07-30 at 1 46 08 pm" src="https://cloud.githubusercontent.com/assets/20569/8990592/65268ace-36c1-11e5-9f1c-6d38180282f3.png">